notebooks/01ingest.py

```diff
@@ -0,0 +1,157 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.11.1
+#   kernelspec:
+#     display_name: 'Python 3.7.10 64-bit (''workflow-miniscope'': conda)'
+#     metadata:
+#       interpreter:
+#         hash: fb69438121b2de1073146525f02c3c959a56b9d2c1d6b0a62c34f01201323d1c
+#     name: python3
+# ---
+
+# # Ingestion routine
+#
+# + The following script outlines the steps to ingest UCLA Miniscope data (acquired metadata and processed data) into the DataJoint `workflow-miniscope`.
+#
+# + To ingest with a completely automated workflow, see `03automate.ipynb`.
+
+import numpy as np
+import os
+os.chdir('..')
+from workflow_miniscope.pipeline import *
+
+# ## Schema diagrams
+#
+# + The following outputs are the diagrams of the schemas comprising this workflow.
+#
+# + Please refer back to these diagrams to visualize the relationships of different tables.
+
+dj.Diagram(lab)
+
+dj.Diagram(subject)
+
+dj.Diagram(session)
+
+dj.Diagram(scan)
+
+dj.Diagram(imaging)
+
+# ## Insert an entry into `subject.Subject`
+
+subject.Subject.insert1(dict(subject='subject1', 
+                             sex='F', 
+                             subject_birth_date='2019-01-01 00:00:01', 
+                             subject_description='no description'))
+
+# ## Insert an entry into `lab.Equipment`
+
+Equipment.insert1(dict(scanner='Miniscope-DAQ-V3'))
+
+# ## Insert an entry into `session.Session`
+
+session.Session.insert1(dict(subject='subject1', 
+                             session_datetime='2021-01-01 00:00:01'))
+
+# ## Insert an entry into `session.SessionDirectory`
+
+session.SessionDirectory.insert1(dict(subject='subject1', 
+                                      session_datetime='2021-01-01 00:00:01', 
+                                      session_dir='<imaging_root_data_dir>/subject1/session0'))
+
+# ## Insert an entry into `scan.Scan`
+
+scan.Scan.insert1(dict(subject='subject1', 
+                       session_datetime='2021-01-01 00:00:01', 
+                       scan_id=0, 
+                       scanner='Miniscope-DAQ-V3', 
+                       acq_software='Miniscope-DAQ-V3',
+                       scan_notes=''))
+
+# ## Populate `scan.ScanInfo`
+#
+# + This imported table stores information about the acquired image (e.g. image dimensions, file paths, etc.).
+
+populate_settings = {'display_progress': True, 'reserve_jobs': False, 'suppress_errors': False}
```

Comment:
should we remove `reserve_jobs` and `suppress_errors`? seems irrelevant to context here. And let's explain what `populate_settings` and `display_progress` do

Reply:
Thanks.  I have removed those two arguments and added explanations.